Automattic/jetpack
# JT: Add disclaimer

- Add a disclaimer to jt-up
- Update docs





#### Changes proposed in this Pull Request:

* Adds some disclaimer to make folks aware of what JT is doing.

#### Jetpack product discussion


pgbkb-7W0-p2#comment-24207 

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* For the CLI change, `jetpack docker jt-up [username] [subdomain]` to see the message.

![2022-01-26 at 4 09 PM](https://user-images.githubusercontent.com/88897/151258129-61ded220-bfcf-450d-aa1c-81762e193ea3.png)
